CONTRIBUTE.md

```diff
@@ -57,9 +57,6 @@ To host the website locally to view the live changes, run the command:
 ```bash
     mkdocs serve
 ```
-Open up http://127.0.0.1:8000/ in your browser, and you'll see the [RAP Community of Practice home page](https://nhsdigital.github.io/rap-community-of-practice/) being displayed with your updates applied.
-
-Read more: [Getting Started with MkDocs](https://www.mkdocs.org/getting-started/#getting-started-with-mkdocs)
```

Comment:
how come we want this removed?

Reply:
oh thanks for spotting, don't know why that disappeared! I've put it back